src/main/java/io/jenkins/plugins/pipelinegraphview/consoleview/PipelineConsoleViewAction.java

```diff
@@ -55,6 +57,7 @@
 public class PipelineConsoleViewAction implements Action, IconSpec {
     public static final long LOG_THRESHOLD = 150 * 1024; // 150KB
     public static final String URL_NAME = "pipeline-overview";
+    public static final int CACHE_AGE = 86400; // 24 hours in seconds
```

Comment:
safer to use time unit https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/TimeUnit.html in case the number is adjusted but the comment isn't